src/main/java/com/bookstore/bookstoreapi/data/repository/BookRepository.java

```diff
@@ -1,8 +1,11 @@
 package com.bookstore.bookstoreapi.data.repository;
 
 import com.bookstore.bookstoreapi.data.entity.Book;
+import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface BookRepository extends JpaRepository<Book, Long> {
   Book findByIsbn(String ISBN);
```

Comment:
Actually this one needs to be a `List<Book>` as well, because I was looking at the `GET request` return and it doesn't return anything but it should return the the whole `Book` object.

I noticed that Intellij is complaining about `ISBN` being named improperly, and that's because uppercasing is only reserved for `final` aka constants, so in every instance where you have `ISBN` refactor it to `isbn`.